docs/source/modules/uq.rst

```diff
@@ -272,3 +286,124 @@ satisfied.
 
 .. seealso::
    See the tutorial for running MCMC in :ref:`tut_mcmc`.
+
+
+
+Bootstrap
+=========
+
+In general, the training dataset contains some random noise. When the data collection
+process is repeated, we will not get the exactly same values, but instead we will get
+(slightly) different values, where the diviation comes from the random noise. If we
+train the model to fit different realizations of the training dataset, we will get a
+distribution of the parameters. The uncertainty of the parameters from this distribution
+gives how the error in the training data is propagated to the uncertainty of the
+parameters. However, often times we don't have the luxury to repeat the data collection.
+A suggestion in this case is to generate artificial datasets and train the model to fit
+these artificial datasets.
+
+.. figure:: ../img/bootstrap.png
+   :alt: Illustration on how bootstrapping works.
+
+Bootstrapping is a way to generate the artificial datasets. We assume that the original
+dataset contains :math:`N` *independent and identically distributed (iid)* data points.
+An artificial, bootstrap dataset is generated by sample :math:`N` points from the original
```

Comment:
Can we change `N` here to `M`? It does not need to be the same size as the original dataset.